### DISCUSSION TITLE
Cannot propose

### DISCUSSION THREAD

**Discussion by zimpha** [snapshot-labs/snapshot#2424] [Q&A]
It shows when I try to create a proposal
<img width="332" alt="image" src="https://user-images.githubusercontent.com/4531711/169880531-51338bb4-d2c1-4979-8616-c10065008a66.png">

I've checked the rpc response from snapshot api, which is

```
{"error":"unauthorized","error_description":"wrong proposal format"}
```

My space is https://snapshot.org/#/test1234567.eth

**Comment by samwhosung** [comment]
@zimpha I just tried and I cannot reproduce. Does it work now for you?

**Reply by zimpha** [reply]
It's still not working, but the status show correctly.
When I first create the space, It showed joined and No members.

<img width="268" alt="image" src="https://user-images.githubusercontent.com/4531711/170414577-48d99709-2cb5-4f43-a01c-e2e7fdd9f46c.png">
